priv/repo/migrations/20201212210836_add_type_to_messages.exs

```diff
@@ -0,0 +1,10 @@
+defmodule ChatApi.Repo.Migrations.AddTypeToMessages do
+  use Ecto.Migration
+
+  def change do
+    alter table(:messages) do
+      add(:type, :string)
+      add(:private, :boolean)
+    end
```

Comment:
any thoughts on adding an index? maybe there's a situation where an agent only wants to see notes they have on a conversation? we could put a compound index on `conversation_id` and `type where type = 'note'`

Reply:
hmm i like the idea but i think for the sake of the initial version let's skip it for now :)